packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue

```diff
@@ -1,19 +1,64 @@
+<!-- eslint-disable vue/no-static-inline-styles -->
+<!-- eslint-disable vue/no-bare-strings-in-template -->
 <!-- eslint-disable vue/no-restricted-class -->
 <template>
-  <editor-content
-    :editor="editor"
-    class="d-rich-text-editor"
-    data-qa="dt-rich-text-editor"
-    v-on="editorListeners"
-  />
+  <div>
+    <!-- why the hell is this visibility: hidden by default??? -->
+    <bubble-menu
+      v-if="editor && link && !hideLinkBubbleMenu"
+      :editor="editor"
+      :should-show="bubbleMenuShouldShow"
+      :tippy-options="tippyOptions"
+      style="visibility: visible;"
+    >
+      <div class="d-popover__dialog">
+        <dt-stack
+          direction="row"
+          gap="200"
+        >
```

Comment:
* Edit link popover needs padding of at least 2, maybe 4. 
* May not need gap at all since all three buttons don't have a border.

Consult with @francotirabasso on that kind of tiny UI detail.

<img width="454" alt="image" src="https://github.com/user-attachments/assets/ed60cb3c-79a2-46bb-bb4d-7b3003d2c303" />

Reply:
@francotirabasso any guidance here?

Reply:
Set to 4 for now, unless suggested otherwise